include/oneapi/dpl/pstl/tuple_impl.h

```diff
@@ -320,15 +331,15 @@ struct tuple<T1, T...>
     // since "template operator=" is not considered as operator= overload
     // the deleted operator= has a preference during lookup
     tuple&
-    operator=(const __internal::tuple<T1, T...>& other) = default;
+    operator=(const tuple<T1, T...>& other) = default;
 
     // for cases when we assign ::std::tuple to __internal::tuple
     template <typename U1, typename... U>
     tuple&
     operator=(const ::std::tuple<U1, U...>& other)
     {
         holder.value = ::std::get<0>(other);
-        next = get_tuple_tail(other);
+        next = oneapi::dpl::__internal::get_tuple_tail(other);
         return *this;
     }
```

Comment:
`operator==` and `operator!=` don't have namespaces for the arguments. And either `swap` and `to_std_tuple` doesn't